content/spin/upgrade.md

```diff
@@ -0,0 +1,86 @@
+title = "Upgrade Spin"
+template = "spin_main"
+date = "2023-05-01T00:01:01Z"
+enable_shortcodes = true
+[extra]
+url = "https://github.com/fermyon/developer/blob/main/content/spin/upgrade.md"
+
+---
+- [Check Spin Version](#check-spin-version)
+- [Upgrade Spin Version](#upgrade-spin-version)
+  - [Upgrade From Source](#upgrade-from-source)
+  - [Upgrading Using Cargo](#upgrading-using-cargo)
+  - [Upgrading Using OS Package Managers](#upgrading-using-os-package-managers)
+- [Upgrade Spin Templates](#upgrade-spin-templates)
+  - [Upgrading Using OS Package Managers](#upgrading-using-os-package-managers-1)
+- [Upgrade Spin Plugins](#upgrade-spin-plugins)
+  - [Upgrading Using OS Package Managers](#upgrading-using-os-package-managers-2)
+
+## Check Spin Version
+
+Your existing version of Spin can be checked using the [Spin Command Line Interface (CLI)](/common/cli-reference) command `spin --version`, as shown below:
+
+<!-- @selectiveCpy -->
+
+```bash
+$ spin --help
+```
+
+You can compare the output of the above command with the version listed by the [latest](https://github.com/fermyon/spin/releases/latest) tag endpoint, in the [Spin GitHub repository](https://github.com/fermyon/spin).
+
+## Upgrade Spin Version
```

Comment:
My first thought was "is this just the binary, can we give them more of a roadmap" but on reflection the different routes are going to upgrade different things.  Like some routes will upgrade "Fermyon core" plugins and others won't.

So I wonder if this is something like "upgrade your Spin distribution", though it feels a bit weird referring to `cargo install` that way...!

Reply:
Thanks, good point.
Perhaps upgrade Spin, then.

Reply:
The word version is implied anyway.